Parsers/VSISDMI.cs

```diff
@@ -1724,296 +1728,1124 @@ public VSISDMI()
+            Identifiers = new List<string>
+            {
+                "230530200",
+                "230535200"
+            },
+            BitFields = new List<BitField>
+            {
+                MMI_M_PACKET,
+                MMI_L_PACKET,
+                MMI_NID_NTC,
+                new BitField
+                {
+                    Name = "MMI_STM_Q_DRIVERINT",
+                    BitFieldType = BitFieldType.Bool,
+                    Length = 1,
+                    Comment = "Need for driver intervention or not during Specific STM Data Entry."
+                },
+                MMI_STM_Q_FOLLOWING,
+                new BitField
+                {
+                    Name = "evc25_spare1",
+                    BitFieldType = BitFieldType.Spare,
+                    Length = 2,
+                    Comment = "Spare for byte alignment"
+                },
+                new BitField
+                {
+                    Name = "evc25_spare2",
+                    BitFieldType = BitFieldType.Spare,
+                    Length = 4,
+                    Comment = "Spare for byte alignment"
+                },
+                MMI_N_ITER,
+                new BitField
+                {
+                    VariableLengthSettings = new VariableLengthSettings
+                    {
+                        Name = "MMI_N_ITER"
+                    },
+                    NestedDataSet = new DataSetDefinition
+                    {
+                        BitFields = new List<BitField>
+                        {
+                            MMI_NID_NTC2,
+                            MMI_STM_NID_DATA,
+                            MMI_EVC_M_XATTRIBUTE,
+                            MMI_L_CAPTION,
+                            MMI_STM_X_CAPTION,
+                            MMI_STM_L_VALUE,
+                            MMI_STM_X_VALUE,
+                            new BitField
+                            {
+                                Name = "MMI_N_ITER2",
+                                BitFieldType = BitFieldType.UInt16,
+                                Length = 16,
+                                Comment =
+                                    "Maximum iteration data pick-up list (range: 0-16).\r\nNote: Higher values are allowed, if a reduced size of caption text is used but limited by the maximum message length."
+                            },
+                            new BitField
+                            {
+                                VariableLengthSettings = new VariableLengthSettings
+                                {
+                                    Name = "MMI_N_ITER2"
+                                },
+                                NestedDataSet = new DataSetDefinition
+                                {
+                                    BitFields = new List<BitField>
+                                    {
+                                        MMI_STM_L_VALUE2,
```

Comment:
The definition of this is the same as MMI_STM_L_VALUE so they should be merged

Reply:
I thought about merging them but I noticed that the comments are slightly different (as per VSIS). I agree the differences are minimal so we can merge them if you like.

MMI_STM_L_VALUE Comment = "Length of X_VALUE for default value (range: 0-10)."
MMI_STM_L_VALUE2 Comment = "Length of X_VALUE for pick-up list value (range: 0-10)."